### PR TITLE
Update message for missing EE for Job pages

### PR DESCRIPTION
**Pull request by nixocio**
Update message for missing EE for Job pages

See: https://github.com/ansible/awx/issues/10355

image

image

### CODE REVIEW

File: awx/ui_next/src/components/JobList/JobListItem.jsx [hunk 1]
```diff
@@ -202,9 +209,29 @@ function JobListItem({
                   dataCy={`job-${job.id}-project`}
                 />
               )}
-              <ExecutionEnvironmentDetail
-                virtualEnvironment={job.custom_virtualenv}
-                executionEnvironment={execution_environment}
+              <Detail
```
Comment (keithjgrant) [inline awx/ui_next/src/components/JobList/JobListItem.jsx]: Is there any reason we can't move this logic/tooltip into the ExecutionEnvironmentDetail and trigger it via a prop? It's seems strange to not use that component, especially when we have to repeat this code again in the JobDetail page

Comment (nixocio) [reply awx/ui_next/src/components/JobList/JobListItem.jsx]: @keithjgrant, I will take a look.

Comment (nixocio) [reply awx/ui_next/src/components/JobList/JobListItem.jsx]: @keithjgrant updated per suggestion.

**Review by mabashian — APPROVED** [review]

**Review by keithjgrant — APPROVED** [review]

**Comment by akus062381** [comment]
https://github.com/ansible/tower-qa/pull/6423

**Review by akus062381 — COMMENTED** [review]
all good!

**Pull request merged**